coralproject/talk
# [CORL-3012]: New commenter badge

## What does this PR do?

These changes add a New commenter plant icon badge next to a commenter's username in the stream when they have joined Coral within the last seven days. It also shows a plant icon badge next to their username in the user history drawer. Also there is a configuration for the new commenter feature in the admin.

## These changes will impact:


## What changes to the GraphQL/Database Schema does this PR introduce?

It adds `newCommenter` to settings and `newCommenter` to a user as well.

## Does this PR introduce any new environment variables or feature flags?

no

## If any indexes were added, were they added to `INDEXES.md`?

n/a

## How do I test this PR?

Sign up a new user. Comment. See the plant icon with tooltip next to their username in the stream. See the plant icon with tooltip next to their username in the user history drawer. Disable the new commenter feature. See that the plant icon shows.

Also check that users who have joined greater than seven days ago do not have the plant icon + tooltip.

## Were any tests migrated to React Testing Library?



## How do we deploy this PR?
